namui/src/namui/common/types/px.rs

```diff
@@ -17,6 +17,12 @@ impl PxExt for f32 {
     }
 }
 
+impl PxExt for i32 {
```

Comment:
I know it's not a good idea to put small patches with other commits but... 😉

Reply:
Ah kk, mot cham ji